alex-becthold/licensing.md

```diff
@@ -0,0 +1 @@
+Simply put, a copyright is the expressed right to use a certain, logo, phrase, image, etc.  For example, the NFL logo is a copyright of the National Football League.  We are affected by copyrights because it limits what we will be able to use to create our game.  We can't use images from another game that is copyrighted, and our game has to be in some way original.  It can't be a copy of another.  In order to stay out of trouble, we need to come up with original ideas for our game and create our own images to use.
```

Comment:
> In order to stay out of trouble, we need to come up with original ideas for our game and create our own images to use.

Checkout https://creativecommons.org/about/ for why you don't have to create your own images for use, if you don't want to.